sniffles/snifflesconfig.py

```diff
@@ -459,7 +466,20 @@ def parse_opt_arg(self, opt, arg):
             self.tcp_ack = True
         # Set percentage of background traffics to be added
         elif opt == "-B":
-            self.background_traffic = int(arg)
+            args = re.split('[:]+', arg)
+            if len(args) == 1:
+                self.background_traffic = int(args[0])
+            else:
+               args = re.split('[,]+', arg)
+               for i in range(0, len(args)):
+                   args_dist = re.split('[:]+', args[i])
+                   if len(args_dist) == 1:
+                       self.usage()
+                   else:
+                       self.background_traffic += int(args_dist[1])
```

Comment:
The first option is probably better as it will impact the original method less.  Overall, we the probability should probably be entirely encompassed by the backgroundtrafficrule such that we just make a call to get the next bg traffic.  Regardless, go with 1 and I will try and remember to set a user story for the second part.